cobalt/build/configs/BUILD.gn

```diff
@@ -24,13 +24,22 @@ config("cobalt_config") {
     configs += [ "//${starboard_path}/platform_configuration" ]
   } else if (is_cobalt_hermetic_build &&
              current_toolchain == cobalt_toolchain) {
-    assert(current_cpu == "x64",
-           "TODO: b/390500840 - $current_cpu not yet supported.")
     configs += [
-      # TODO: b/390500840 - Remove hardcoded x64 config, switch config based on platform.
       "//starboard/build/config/modular/x64",
       "//starboard/build/config:starboard",
     ]
+    if (target_cpu == "arm") {
```

Comment:
Couple of notes on this section
1. You'll likely need to remove the x64 config added above
2. Use current_cpu over target_cpu whenever possible (you can do so here)
3. This'll need some reconfiguration to be more general, which I can handle in my work. Because partners will build with an evergreen config, they shouldn't control the configuration here.